### ISSUE TITLE
Update ethereum-alarm-clock.com

### ISSUE THREAD

**Issue by pipermerriam** [ethereum-alarm-clock/eac.js#11]
If you guys would like to update the main website at some point lets talk about how we can do that.  Currently it's a django app running on heroku.

**Comment by lsaether** [comment]
Yeah I think we would like to update it at some point. I'm not super familiar with how Heroku builds, but would it be possible to host the website here on the organization and be able to add a webhook to build on update? I think that might be the best way to do it.

**Comment by pipermerriam** [comment]
Yes, using gh-pages and hosting it using this organization would be fine with me.

I'm game to point the DNS wherever you'd like to host it.
